x-pack/packages/security-solution/common/src/flyout/common/components/flyout_error.test.tsx

```diff
@@ -9,7 +9,7 @@ import React from 'react';
 import { __IntlProvider as IntlProvider } from '@kbn/i18n-react';
 import { render } from '@testing-library/react';
 import { FlyoutError } from './flyout_error';
-import { FLYOUT_ERROR_TEST_ID } from './test_ids';
+import { FLYOUT_ERROR_TEST_ID } from '../test_ids';
```

Comment:
just curious about why moving the `test_ids` file outside of the `components` folder like it was in the Security Solution application?

Reply:
There was `test_id` file in `./shared/components/` and `./shared/` so i took decision that `test_id` file actually belong outside components folder because it is not a component. But let me know if you have strong opinion on this.

I thing both of these `test_id` files can be combined.